Project-Aurora/Project-Aurora/Devices/YeeLight/YeeLightDevice.cs

```diff
@@ -79,8 +79,6 @@ public bool IsPeripheralConnected()
 
         public bool Reconnect()
         {
-            light.CloseConnection();
-
```

Comment:
I don't understand why "light.CloseConnection();" is not needed, because if light is connected then nothing going to close it. So the isConnected = false;  value don't going to change in Connect(), because light.IsConnected() is true.